packages/agent-toolkit/src/core/tools/platform-api-tools/create-group/create-group-tool.ts

```diff
@@ -0,0 +1,52 @@
+import { z } from 'zod';
+import {
+  PositionRelative,
+  CreateGroupMutation,
+  CreateGroupMutationVariables,
+} from '../../../../monday-graphql/generated/graphql';
+import { createGroup } from './create-group.graphql';
+import { ToolInputType, ToolOutputType, ToolType } from '../../../tool';
+import { BaseMondayApiTool, createMondayApiAnnotations } from '../base-monday-api-tool';
+
+export const createGroupToolSchema = {
+  boardId: z.string().describe('The ID of the board to create the group in'),
+  groupName: z.string().max(255).describe('The name of the new group (maximum 255 characters)'),
+  groupColor: z.string().optional().describe('The HEX code color for the group (e.g., #ff642e)'),
+  relativeTo: z.string().optional().describe('The ID of the group to position this new group relative to'),
```

Comment:
do we have query to get groups? how would the MCP know what to do here?

Reply:
A user would need to provide that group ID, something like "Create a group called XYZ and place it right after the group ABC"

Reply:
hm.. you might be right. I can just remove those params (relativeTo, positionRelativeMethod). @shaneer what do you think?